src/components/price/service.ts

```diff
@@ -36,21 +44,26 @@ export class PriceService {
                 prices = await this.priceProvider.getPrices(AppConfig.PRICE.COINS);
             }
 
+            let supportedPrices = {};
+            for (const pair in this.userConfig.PAIRS) {
+                if (prices[pair]) {
+                    supportedPrices[pair] = prices[pair];
+                } else {
+                    logError(`SUPPORTED_PAIR_MISSING_PRICE: ${pair}`);
+                }
+            }
+
             if (Object.values(prices).length > 0) {
-                this.setPrices(prices);
-                this.setPricesWithSpreadAndFee(prices);
+                this.setPrices(supportedPrices);
+                this.setPricesWithSpreadAndFee(supportedPrices);
             }
         } catch (err) {
-            logError('PRICE_SERVICE_DOWN', err);
-
-            if (AppConfig.PRICE.USE_FALLBACK && PriceProviders[AppConfig.PRICE.FALLBACK]) {
-                logInfo(`Starting fallback price service: ${AppConfig.PRICE.FALLBACK}`);
-                this.priceProvider = new PriceProviders[AppConfig.PRICE.PROVIDER]();
-                this.fallbackProvider = new PriceProviders[AppConfig.PRICE.FALLBACK]();
-                await this.update(this.fallbackProvider);
-            } else {
-                // logInfo('Liquidity Node is Shutting Down - No fallback porice provider.......');
-                // process.exit(1);
+            console.log(err);
+            if (maxTries > 0) {
```

Comment:
What happens when maxtries <=0 ?
Fallback price provider is never used. 
Did we removed that option?
If so, this label should be removed `ogInfo(`Starting fallback price service: ${this.userConfig.PRICE.PROVIDER}`);`